f/export/postgres_to_geojson/postgres_to_geojson.py

```
@@ -0,0 +1,136 @@
+        data (tuple): A tuple containing columns and rows fetched from the database.
+
+    Returns
+    -------
+        dict: A GeoJSON FeatureCollection with features extracted from the data.
+    """
+
+    columns, rows = data
+    features = []
+    for row in rows:
+        properties = {}
+        geometry = {}
+        feature_id = None
+
+        # The expected schema here is that geometry columns are prefixed with "g__"
+        # If an "_id" column is present, it is used as the feature id
+        # All other columns are treated as properties
+        for col, value in zip(columns, row):
+            if col == "_id":
+                feature_id = value
+            elif col.startswith("g__"):
+                geometry[col[3:]] = value
+            else:
+                properties[col] = value
+
+        feature = {
+            "type": "Feature",
+            "id": feature_id,
+            "properties": properties,
+            "geometry": geometry,
+        }
+        features.append(feature)
+
+    feature_collection = {
+        "type": "FeatureCollection",
+        "features": features,
+    }
+
+    logger.info(f"GeoJSON created with {len(features)} features")
+
+    return feature_collection
+
+
+def save_file(data, db_table_name: str, storage_path: str):
+    """
+    Saves the provided data as a GeoJSON file in the specified storage path.
+
+    Parameters
+    ----------
+        data (dict): The data to be saved, formatted as a GeoJSON FeatureCollection.
+        db_table_name (str): The name of the database table, used to name the output file.
+        storage_path (str): The directory path where the GeoJSON file will be saved.
+    """
+
+    try:
+        os.makedirs(storage_path, exist_ok=True)
+        geojson_path = os.path.join(storage_path, f"{db_table_name}.geojson")
+        with open(geojson_path, "w") as f:
+            json.dump(data, f)
+    except OSError as e:
+        logger.error(f"Error saving file {geojson_path}: {e}")
```

Do you really want to swallow this exception?  As-is I think the script will succeed after failing to write the file.

I'd drop the previous 2 lines.  If there's an OSError, the script will fail, which seems more like what we want.